openstef_dbc/services/write.py

```diff
@@ -363,6 +364,8 @@ def write_kpi(self, pj, kpis):
         # add date
         df = df.set_index("date")
 
+        # Convert nan / inf since these are not supported in influx
+        df = df.replace([np.inf, -np.inf], np.nan)
         df = df.fillna(value=0)
```

Comment:
I agree with changing np.inf to np.nan. 
Another point: Can we change the 0 to something like -1 as 0 could technically be a correct value for most metrics whereas -1 is not possible in MAE, rMAE, ... etc. What do you think?

Reply:
I agree, I'll change +/- info to +/- a large number instead